apps/emqx_schema_registry/src/emqx_schema_registry_schema.erl

```diff
@@ -111,7 +111,15 @@ fields(avro) ->
 fields(protobuf) ->
     [
         {type, mk(?protobuf, #{required => true, desc => ?DESC("schema_type_protobuf")})}
-        | common_fields(binary())
+        | common_fields(hoconsc:union([binary(), ref(protobuf_bundle_source)]))
+    ];
+fields(protobuf_bundle_source) ->
+    [
+        {type, mk(bundle, #{required => true, desc => ?DESC("protobuf_source_bundle_type")})},
+        {root_proto_path,
```

Comment:
I like it.
I think we should aim to convert other schema sources to files too, so to reference the name in cluster.hocon.
this makes below things slightly more complicated though:

1. initial copy of the directories https://github.com/emqx/emqx/blob/08562c47a7c71d391526baba6d5ea8f38c1bac9d/apps/emqx_conf/src/emqx_conf_app.erl#L72
2. data backup(export) and import

Reply:
yes, new node will sort of "rebase"